src/test/java/com/spanish_inquisition/battleship/common/AppLoggerTest.java

```diff
@@ -0,0 +1,25 @@
+package com.spanish_inquisition.battleship.common;
+
+import org.testng.annotations.Test;
+
+import java.util.logging.Level;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertNull;
+
+/**
+ * @author Michal_Partacz
+ */
+public class AppLoggerTest {
+
+    @Test
+    public void testInitializeLogger() throws Exception {
```

Comment:
it's good to write what is expected, in method name or with //given //when //then